xc/xenops_server_xen.ml

```diff
@@ -2940,15 +2940,11 @@ module VIF = struct
               let open Xenops_interface.Pci in
               let pci_state = PCI.get_state' vm pci in
               let open Xenops_interface.Pci in
-              begin match pci_state.plugged with
-              | true ->
-                { unplugged_vif with
-                  Vif.active = true;
-                  plugged = true;
-                  device = Some device;
-                }
-              | false -> unplugged_vif
-              end
+              { unplugged_vif with
+                Vif.active = get_active vm vif;
+                plugged = pci_state.plugged;
+                device = Some device;
```

Comment:
Does it matter if we set `device` to `Some device` when `pci_state.plugged` is `false`?

Reply:
This vif device is there already, even its backend VF is not plugged.